Assets/Scripts/Characters/Player/CollectableItems.cs

```diff
@@ -0,0 +1,54 @@
+﻿
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CollectableItems : MonoBehaviour
+{
+    public Inventory Inventory1;
```

Comment:
```suggestion
    private Inventory Inventory1;
```
В функции Start() он берется у персонажа, так что нет смысла держать его пабликом?